script.sh

```diff
@@ -0,0 +1,11 @@
+VM_ID=$1
```

Comment:
Please add a shebang:
```suggestion
#!/bin/bash
VM_ID=$1
```